### DISCUSSION TITLE
Rename "Properties" to "EP Properties"

### DISCUSSION THREAD

**Discussion by cheeseandjamsandwich** [valinet/ExplorerPatcher#3832] [Ideas]
To make things clearer, could ExplorerPatcher's "Properties" option in the menus be renamed to "EP Properties", or even "ExplorerPatcher Properties"?

This would be very useful for new users, and for our family/friends that we've helped get a useable taskbar back on their machines, with the help of ExplorerPatcher.  It'd be easier for them to find the properties window themselves, and easier for us to guid them there.

"Properties" is used all over the place.  Having it as "EP Properties" makes it certain what you're clicking on, that you've found the options/properties for ExplorerPatcher.

**Comment by pyrates999** [comment]
Good idea :)
